src/pages/Chat/index.jsx

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
can you move this to each line were its a error. Disabling all the file is a bit extreme

Reply:
thanks